include/irr/asset/filters/CSummedAreaTableImageFilter.h

```diff
@@ -93,10 +115,13 @@ class CSummedAreaTableImageFilter : public CMatchedSizeInOutImageFilterCommon, p
 			if (state->scratchMemoryByteSize < state_type::getRequiredScratchByteSize(state->inImage))
 				return false;
 
+			if (state->inMipLevel != state->outMipLevel)
```

Comment:
nope, just check the size (I think the base class `validate` method already does this)

Reply:
I don't see that the base checks it

Reply:
the validation is needed here - solved